config/justask.yml.example

```diff
@@ -59,3 +59,12 @@ hcaptcha:
 
 # TOTP Drift period in seconds
 otp_drift_period: 30
+
+accepted_image_formats:
```

Comment:
please add a comment to this describing what it's used for

Reply:
a38c1342